modules/ui/ConceptWindow.py

```diff
@@ -99,15 +99,20 @@ def __general_tab(self, master, concept: ConceptConfig):
                          tooltip="Enable or disable this concept")
         components.switch(frame, 1, 1, self.ui_state, "enabled")
 
+        # validation_images
+        components.label(frame, 2, 0, "Validation images",
```

Comment:
"validation images" isn't great as a name. Maybe "Validation Concept" would be better. You aren't just using the images, but also the prompts (and maybe other data for future models)